SECP-service/src/main/resources/assets/app/views/home/home.html

```diff
@@ -0,0 +1,138 @@
+<header class="masthead">
+    <div class="container h-100">
+        <div class="row h-100">
+            <div id="mastdiv" class="col-lg-7 my-auto">
+                <div class="header-content mx-auto">
+                    <h1>SECP</h1>
+                    <h3 id="introblurb">Mitigate corporate espionage within your organization via secure communication channels and user auditing capabilities.</h3>
+                    <a href="#/login" class="btn btn-outline btn-xl js-scroll-trigger">Login</a>
+                    <a href="#/register" class="btn btn-outline btn-xl js-scroll-trigger">Register</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</header>
+<section class="features" id="features">
+    <div class="container">
+        <div class="section-heading text-center">
+            <h2>More Than Just Another Chat App</h2>
+            <p class="text-muted">Check out what we can do for your organization!</p>
```

Comment:
Once the major features of the app are good , we can add a video here showing some of the stuffs the app is capable of. :)

Reply:
Agreed, apparently we will have to make a video for the project so we can always publish it on Youtube then embed it here when we are ready 👍